### ISSUE TITLE
Help for Remove Service incorrectly identifies optional arguments

### ISSUE THREAD

**Issue by taraskozub-percona** [percona/pmm#1242]
### Description

**Impact on the user**
Unable to correctly identify the necessary arguments ahead of execution

**Workaround**
Trial and error at the command line

**Details**
The help information for removing a service is inaccurate and does not help the user successfully execute the action first time without prior knowledge of additional information.

### Expected Results

The help information is sufficient to allow the user to successfully identify the arguments required to execute the action.

### Actual Results

Help information identifies both the flags and the service-name as optional arguments. In addition, the help information does not state anything useful alongside the service-id option to help clarify matters. Without extra knowledge, the removal will fail due to missing either the service-name or service-id

### Version

client 2.28

### Steps to reproduce

Register a new PMM agent using pmm-agent setup, or similar
Add a single service (mysql, mongodb, postgresql, proxysql, haproxy, external) using pmm-admin add of one or more types, maintaining a 1:1 ratio of instance:service-type
Use pmm-admin remove --help to identify the necessary arguments to remove the service
Remove one of the services with pmm-admin remove <service-type>

### Code of Conduct

- [X] I agree to follow Percona Community Code of Conduct

**Comment by daniel-shuy** [comment]
I'm unable to reproduce this. If there is only 1 instance of the service type (mysql, mongodb, postgresql, proxysql, haproxy, external), then `pmm-admin remove <service-type>`  works without having to specify service-name or service-id

**Comment by michal-kralik** [comment]
@daniel-shuy The description was inaccurate. Due to recent improvements the command did not fail since we automatically lookup the single service type.

I just updated the description - you need to add at least two services of the same type to replicate the issue.
Feel free to reach back out in case you need any help.

Happy Hacktoberfest!

**Comment by daniel-shuy** [comment]
@michal-kralik Thanks!

Personally I found the existing help good enough, the service-name or service-id can be easily found by running `pmm-admin list`.

The only improvement I can think of is to explicitly specify that service-name or service-id is required if there are multiple services with the given service-type, and that the service-name or service-id can be found by running `pmm-admin list`. What do you think?

**Comment by michal-kralik** [comment]
you're right. With the recent changes the help is improved and this is no longer an issue.
I'll close this.

Apologies should this cause you to lose a Hacktoberfest contribution - this obviously wasn't my intention :disappointed:

**Comment by daniel-shuy** [comment]
No worries, glad to help!
